save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/ExecutionController.kt

```diff
@@ -140,9 +140,8 @@ class ExecutionController(private val executionService: ExecutionService,
             "Can't rerun execution $id, project ${execution.project.name} has no associated git address"
         }
         val testRootPath = if (executionType == ExecutionType.GIT) {
-            execution.testSuiteIds?.let {
-                require(it == "ALL") { "Only executions with \"ALL\" tests suites from a GIT project are supported now" }
-                testSuitesService.findTestSuitesByProject(execution.project)
+            execution.testSuiteIds?.split(", ")?.map {
+                testSuitesService.findTestSuiteById(it.toLong()).get()
```

Comment:
Please add `orElseThrow` with description, which IDs don't exists. If we accidentally clean up something and it will simply start failing here, it'll be hard to debug.

Reply:
`get` method throws exceptions by itself in case when there is no such element,
however I added `orElseThrow`, it will be easy to find error by our message